packages/ra-core/src/auth/useCanAccessResources.ts

```diff
@@ -0,0 +1,148 @@
+import { useEffect } from 'react';
+import {
+    useQueries,
+    UseQueryOptions,
+    UseQueryResult,
+} from '@tanstack/react-query';
+import useAuthProvider from './useAuthProvider';
+import useLogoutIfAccessDenied from './useLogoutIfAccessDenied';
+import { useEvent } from '../util';
+
+/**
+ * Checks whether users can access the provided resources (resource in the RBAC sense).
```

Comment:
Remove the RBAC mention

Reply:
How would you suggest to make the distinction with React-Admin resources

Reply:
"resource in the access control sense"?